addons/account/models/account_move.py

```diff
@@ -2918,6 +2918,37 @@ def is_internal_partner(partner):
         move.message_subscribe(list(all_followers_ids))
         return move
 
+    def _post_check_non_negative(self):
+        """It checks that the total amount of a normal account cannot be less than zero.
+
+        In some accounting localisations this should be adjusted/disabled.
+        """
+        self.ensure_one()
```

Comment:
I'm not a big fan of the multiple `self.ensure_one()`s but at the same time, there is no real problem with it. If we have a loop, the method could be called easier on a batch of invoices.

Reply:
ok, i rewrite them to be multi